index.html

```diff
@@ -497,18 +498,18 @@ <h3>The <dfn>observe()</dfn> method</h3>
         </li>
         <li>
           If |document| is not null and is not [=allowed to use=] the [=policy-controlled
-          feature=] named "`compute-pressure`", throw {{NotAllowedError}}.
+          feature=] named "<a>compute-pressure</a>", throw {{NotAllowedError}}.
```

Comment:
same nit about linking to the powerful feature for permissions-policy integration.

Reply:
See above